kubernetes_asyncio/config/incluster_config.py

```diff
@@ -35,11 +36,16 @@ def _join_host_port(host, port):
 
 class InClusterConfigLoader(object):
 
-    def __init__(self, token_filename,
-                 cert_filename, environ=os.environ):
+    def __init__(self,
+                 token_filename,
+                 cert_filename,
+                 try_refresh_token=True,
+                 environ=os.environ):
         self._token_filename = token_filename
         self._cert_filename = cert_filename
         self._environ = environ
+        self._try_refresh_token = try_refresh_token
+        self._token_refresh_period = datetime.timedelta(minutes=1)
```

Comment:
Could you use module constant instead?

Reply:
Sure, I will include that change shortly. The initial approach was derived from https://github.com/kubernetes-client/python/blob/master/kubernetes/base/config/incluster_config.py#L47, to keep them similar (since this module was derived from that one), but I don't have a strong preference.

Reply:
Oh, sorry. I wasn't aware of it. However, thanks for changing it.